app/controllers/api/v1/mixins/service_offering_mixin.rb

```diff
@@ -9,6 +9,8 @@ def service_offering_check(order)
           if service_offering_service.archived
             raise Catalog::ServiceOfferingArchived, "Service offering for order #{order.id} has been archived and can no longer be ordered"
           end
+
+          return true
```

Comment:
Do we need `return` here?

Reply:
Yes, because the `submit_order?` policy definition needs to be either true/false.